packages/11ty/content/_computed/eleventyComputed.js

```diff
@@ -95,7 +95,9 @@ module.exports = {
     if (!object || !object.length) return
     return object
       .reduce((validObjects, item) => {
-        const objectData = objects.object_list.find(({ id }) => id === item.id)
+        const objectData = objects.object_list && objects.object_list.length
+          ? objects.object_list.find(({ id }) => id === item.id)
+          : item
```

Comment:
This change is to account for object data referenced in full in front matter (see https://github.com/thegetty/quire-starter-objects-test)

```yaml
title: "Almanac"
aliases: [almanacs]
layout: page
order: 101
tags: thing
object:
  - name: "Almanac"
    thumbnail: "figures/things/almanac.jpg"
    owner:
      - first_name: Claude-Joseph
        last_name: Vernet
        years: 1714–89
    type: [Instrument]
    theme: [Everyday, Louvre, Travel]
    material: [Synthetic Materials | Paper]
mentions: [watch, order book, journal, snuffbox, sword, umbrella, wig]
```